addons/backgrounds/src/BackgroundPanel.js

```diff
@@ -130,8 +129,11 @@ export default class BackgroundPanel extends Component {
   }
 
   render() {
+    const { active } = this.props;
     const backgrounds = [...this.state.backgrounds];
-
+    if (!active) {
+      return null;
+    }
     if (!backgrounds.length) return <Instructions />;
```

Comment:
Newline between statements, and curly braces? Not sure what the rules here are.

Reply:
If eslint passes then it's OK